src/main/java/data/SpinOff.java

```diff
@@ -0,0 +1,14 @@
+
+public class SpinOff extends Recipe {
+
+  private Recipe parent;
```

Comment:
Parent makes sense when thinking about subclasses, but imo it's not the best descriptor for their actual relationship. Maybe "original", instead?